sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilterSuite.scala

```diff
@@ -2228,6 +2228,50 @@ abstract class ParquetFilterSuite extends QueryTest with ParquetTest with Shared
       }
     }
   }
+
+  test("disable filter pushdown for collated strings") {
```

Comment:
I think this PR does not target parquet only, but all file sources. Shall we put the test in `FileBasedDataSourceSuite`?